esdt-safe/src/lib.rs

```diff
@@ -229,7 +252,11 @@ pub trait EsdtSafe:
     /// fee_amount = price_per_gas_unit * eth_tx_gas_limit
     #[payable("*")]
     #[endpoint(createTransaction)]
-    fn create_transaction(&self, to: EthAddress<Self::Api>) {
+    fn create_transaction(
+        &self,
+        to: EthAddress<Self::Api>,
+        opt_refund_address: OptionalValue<ManagedAddress>,
```

Comment:
ManagedOption  or Option?

Reply:
We use ManagedOption for struct fields due to encoding.